docs/docs/08-references/01-cli.md

```diff
@@ -2837,7 +2837,7 @@ Querying subcommands
 **Options inherited from parent commands**
 
 ```
-      --node string   <host>:<port> to tendermint rpc interface for this chain (default "https://rpc.cosmos.network:443")
+      --node string   <host>:<port> to tendermint rpc interface for this chain (default "https://rpc.cosmos.directory/cosmoshub:443")
```

Comment:
Is this notation right?

Reply:
I wonder how this happened given that I searched and replaced 😅 
but this is indeed incorrect.